client/consensus/aura/src/lib.rs

```diff
@@ -708,7 +804,7 @@ pub fn import_queue<B, C, P, T>(
 	};
 	Ok(BasicQueue::new(
 		verifier,
-		block_import,
+		Box::new(AuraBlockImport::<_, _, _, P>::new(block_import, client)),
```

Comment:
The `AuraBlockImport` is only used in the `ImportQueue`, but isn't being used in `start_aura`. Usually, we have the caller instantiate the `BlockImport` outside of the consensus crate to avoid this issue.

Reply:
Not sure what are you proposing to change

Reply:
The `AuraBlockImport `should be created outside this function, in the service

Reply:
@rphmeier  Now when I dug into some of the service code, it starts to make sense (for me), thanks!